Fidela1/my-brand
# Implementing landing page and creation of dashboard

#### What does this PR do?

 this PR implement landing page

#### Description of Task to be completed?

- [x] adding landing page
- [x] creation of dashboard

#### How should this be manually tested?

 clone to the repo

 Checkout branch feat-implement-landing-page-JsG1NTTl
 go to the index.html to see landing page
 go on login and click on login button to see dashboard

Trello https://trello.com/b/AGxbCKRC/capstone-project
